apis/config/v1beta1/configuration_types.go

```diff
@@ -169,6 +169,22 @@ type ControllerMetrics struct {
 	// metrics will be reported.
 	// +optional
 	EnableClusterQueueResources bool `json:"enableClusterQueueResources,omitempty"`
+
+	// Configuration for exposing LocalQueue Metrics
+	// +optional
+	LocalQueueMetrics *LocalQueueMetrics `json:"localQueueMetrics,omitempty"`
+}
+
+type LocalQueueMetrics struct {
+	// Allows admin to disable LocalQueueMetrics without altering defined selectors
+	// defaults to true
```

Comment:
Agree with https://github.com/kubernetes-sigs/kueue/pull/4581/files#r1993449796.

Given https://github.com/kubernetes-sigs/kueue/pull/4581/files#r1994044295, I don't think we should enable by default.

I think this change is good and we should allow for configuration but I don't think the default should be on for everyone.

Reply:
wdyt @kannon92 ?

Reply:
Yea, that sounds good to me. In openshift, we actually have a policy to forbid booleans as they never age well.